### ISSUE TITLE
Did the tools support windows 7

### ISSUE THREAD

**Issue by wjc-x** [HostsTools/Windows#40]
When my friend uses this tools on windows 7, all tool just flash past.

**Comment by ghost** [comment]
> OS type | Minimun Supported Version
> --------|-------------------
> WorkStation | Microsoft Windows XP Family
> Server | Microsoft Windows Server 2003 Family

**Comment by wjc-x** [comment]
Yeah, I saw the OS support type. But, it seems like it not support Windows 7 64-bit, the Windows 7 32-bit can run the software well.

**Comment by wjc-x** [comment]
Does the author have the plan to complete the tools for Windows all platform?

**Comment by fjzzq2002** [comment]
Works well so far in my Win7 64bit.

**Comment by fjzzq2002** [comment]
Works well so far on my win7 64bit :p

**Comment by ghost** [comment]
@wjc-x  Coding environment is Windows 7 Ultimate 64bit, Full test accept in Windows XP, Windows 7 64bit, Windows 8.1 Pro 64bit.

**Comment by lihui327** [comment]
> 当我的朋友在Windows 7上使用此工具时，所有工具刚刚闪过。

似乎我也不行
